crawl-ref/source/describe-god.cc

```diff
@@ -1022,23 +1106,36 @@ static void build_partial_god_ui(god_type which_god, shared_ptr<ui::Popup>& popu
 # define MORE_PREFIX "[<w>!</w>/<w>^</w>" "]: "
 #endif
 
-    const char* mores[3] = {
-        MORE_PREFIX "<w>Overview</w>|Powers|Wrath",
-        MORE_PREFIX "Overview|<w>Powers</w>|Wrath",
-        MORE_PREFIX "Overview|Powers|<w>Wrath</w>",
+    int mores_index = descs[3].empty() ? 0 : 1;
+    const char* mores[2][4] =
+    {
+        {
+            MORE_PREFIX "<w>Overview</w>|Powers|Wrath",
+            MORE_PREFIX "Overview|<w>Powers</w>|Wrath",
+            MORE_PREFIX "Overview|Powers|<w>Wrath</w>",
+            MORE_PREFIX "Overview|Powers|Wrath"
+        },
+        {
+            MORE_PREFIX "<w>Overview</w>|Powers|Wrath|Extra",
+            MORE_PREFIX "Overview|<w>Powers</w>|Wrath|Extra",
+            MORE_PREFIX "Overview|Powers|<w>Wrath</w>|Extra",
+            MORE_PREFIX "Overview|Powers|Wrath|<w>Extra</w>"
```

Comment:
we should eventually convert this to use a widget, aidainh wrote one for this kind of menu. This may actually be the easiest way to get it working on webtiles.

Reply:
Converting it to a widget might also be the easiest way to make the pane's name depend on the god. I'm still not super familiar with the ui code in terms of the set of features available.

Reply:
I think the existence of this widget was more wishful thinking than reality -- maybe aidanh and I talked about it. But it doesn't seem to actually exist.